pkg/config/setup/system_probe.go

```diff
@@ -181,6 +181,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 
 	cfg.BindEnvAndSetDefault(join(spNS, "max_tracked_connections"), 65536)
 	cfg.BindEnv(join(spNS, "max_closed_connections_buffered"))
+	cfg.BindEnv(join(spNS, "max_failed_connections_buffered"))
```

Comment:
why those values are under system-probe configuration and not network_config?

Reply:
I'm happy to move these. this config is very similar to the existing one for closed connections, which is why I put them together

Reply:
I moved this config to `netNS`